frontend/package.json

```diff
@@ -18,6 +18,7 @@
     "@vueuse/core": "^7.0.1",
     "autoprefixer": "^10.2.6",
     "axios": "^0.21.2",
+    "dayjs": "^1.10.7",
```

Comment:
So are we going to move off momentjs and use dayjs in the future ? cc @LiuJi-Jim

Reply:
the dayjs have the same API of moment.js and reduce its bundle size by 97%